isofit/test/test_cli.py

```diff
@@ -82,22 +87,26 @@ def files(cube_example):
 
 # fmt: off
 @pytest.mark.parametrize("args", [
-    ["ang", "--presolve", 1, "--emulator_base", EMULATOR_PATH, "--n_cores", CORES, "--analytical_line", 1, "-nn", 10, "-nn", 50,],
-    ["ang", "--presolve", 1, "--emulator_base", EMULATOR_PATH, "--n_cores", CORES, "--analytical_line", 1, "-nn", 10, "-nn", 50, "-nn", 10, "--pressure_elevation",],
-    ["ang", "--presolve", 1, "--emulator_base", EMULATOR_PATH, "--n_cores", CORES, "--empirical_line", 1, "--surface_category", "additive_glint_surface",],
```

Comment:
These are boolean flags and do not accept a parameter.

Reply:
This was a breaking change introduced in #441, the revision is correct for dev.